index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/ea73301412.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <!-- Home -->
+    <section class="home bg-home">
+        <!-- HEADER -->
+        <header>
+            <div class="top-bar">
+                <ul class="top-bar-list">
+                    <li class="topbar-item"><a href=""><i class="fa-brands fa-facebook-f"></i></a></i></li>
+                    <li class="topbar-item"><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                    <li class="topbar-item"><a href="">English</a></li>
+                    <li class="topbar-item"><a href="">My Page</a></li>
+                    <li class="topbar-item"><a href="">Logout</a></li>
+                </ul>
+            </div>
+            <nav class="menu">
+                <button class="hamb-icon">
+                    <i class="fa-sharp fa-solid fa-bars"></i>
+                </button>
```

Comment:
- [ ] Please fix your mobile navigation menu because it seems like your mobile menu navigation doesn't have the background colour and because of the transparent background, your design looks unprofessional. Kindly take a look at the below-attached video to see the problem.

### Your Mobile Design Demo

[Screencast from 08-07-23 04:01:35 PM IST.webm](https://github.com/ulises2607/cayafate-project/assets/66476329/161c0ce8-473a-4872-b946-ed52d85f8b1a)


### Your mobile menu should look like

[Screencast from 08-07-23 04:02:59 PM IST.webm](https://github.com/ulises2607/cayafate-project/assets/66476329/6f8b9572-0bca-4008-80c6-1d739fca71aa)